cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,37 @@
+import {MenuContentPage, ProductListPage, ShoppingCartPage, LoginPage} from "../page/index";
+import {AddressStepPage, ShippingStepPage, PaymentStepPage} from "../page/index";
```

Comment:
you can rewrite this as:
```suggestion
import {
  MenuContentPage,
  ProductListPage,
  ShoppingCartPage,
  LoginPage,
  AddressStepPage,
  ShippingStepPage,
  PaymentStepPage
} from "../page/index";
```